spec/app/domain/authentication/constraints/non_permitted_constraint_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe(Authentication::Constraints::NonPermittedConstraint) do
+  let(:non_permitted_constraint) { ["iat", "nbf", "exp", "iss"] }
+  let(:raised_error) { ::Errors::Authentication::Constraints::NonPermittedRestrictionGiven }
+
+  subject(:constraint) do
+    Authentication::Constraints::NonPermittedConstraint.new(non_permitted: non_permitted_constraint)
+  end
+
+  context "when validating empty array" do
+    subject do
+      constraint.validate(resource_restrictions: [])
+    end
+
+    it "raises an error" do
```

Comment:
why?

Reply:
Good catch thanks

Reply:
The title needs to be change to not raise_error i put wrong title

Reply:
I forgot to change titles